client/src/templates/Challenges/components/completion-modal-body.tsx

```diff
@@ -113,6 +114,13 @@ export class CompletionModalBody extends PureComponent<
               </div>
             </div>
           </div>
+          <div>
+            <h3>
+              {t('learn.project-complete', {
```

Comment:
Here I am not able to translate the output. It is printing `learn.project-complete` as is.

![image](https://user-images.githubusercontent.com/28809492/182011773-37ed7469-2c33-45e8-9105-d4f8d7e9dc0a.png)
###

Reply:
ya that happened, because a change was made before the function re-renders. so you got string instead of normal value